### PR TITLE
Feat: add navbar

### PR DESCRIPTION
**Pull request by F-Kirchhoff**
Added a Responsive Navbar based on Flexbox

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,7 +8,62 @@
...
+          w-full
+          py-4
+          px-2
+        "
+      >
+        <h1 class="w-full h-5 relative text-2xl font-bold">
+          <span class="absolute right-1/2 mr-20">phi</span>
+          <span class="absolute left-1/2 ml-20">space</span>
+        </h1>
+        <div class="w-full h-0.5 my-4 bg-gray-500"></div>
```
Comment (PReichetanz) [inline index.html:30]: Did you add this div for later use or just as help for positioning?

File: index.html [hunk 2]
```diff
@@ -8,7 +8,62 @@
...
+        <div class="w-full relative -top-4">
```
Comment (PReichetanz) [inline index.html:31]: I have an idea how the nav will look after wrapping, but why exactly did you split it into to lists?

**Review by PReichetanz — APPROVED** [review]
Nice work, my comments are rather questions for my understanding than hints for a better solution.

File: index.html [hunk 3]
```diff
@@ -8,7 +8,62 @@
       href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css"
       rel="stylesheet"
     />
-    <title>Document</title>
+    <title>Phispace</title>
   </head>
-  <body></body>
+  <body class="bg-gray-800 text-purple-50">
+    <div class="container flex flex-col items-center w-full mx-auto">
```
Comment (badabam) [inline index.html:14]: You could put those classes to the `body` as well, couldn't you?

File: index.html [hunk 4]
```diff
@@ -8,7 +8,62 @@
...
+          items-center
+          w-full
+          py-4
+          px-2
+        "
+      >
+        <h1 class="w-full h-5 relative text-2xl font-bold">
+          <span class="absolute right-1/2 mr-20">phi</span>
+          <span class="absolute left-1/2 ml-20">space</span>
+        </h1>
```
Comment (badabam) [inline index.html:29]: I wouldn't put the `h1` into the `nav`, but it's not necessarily wrong.

File: index.html [hunk 5]
```diff
@@ -8,7 +8,62 @@
...
+        <div class="w-full h-0.5 my-4 bg-gray-500"></div>
+        <div class="w-full relative -top-4">
+          <ul
+            class="
+              mr-16
+              flex flex-wrap
+              justify-start
+              absolute
```
Comment (badabam) [inline index.html:37]: I think you don't need to position your `ul`s with `absolute`. You could give the parent a `flex` class and justify them.

File: index.html [hunk 6]
```diff
@@ -8,7 +8,62 @@
...
+              left-0
+              mr-20
+            "
+          >
+            <li class="text-xl m-1"><a href="#">contact</a></li>
+            <li class="text-xl m-1"><a href="#">about</a></li>
+          </ul>
+        </div>
+        <img
+          class="absolute top-1 w-24 bg-gray-800 p-2 mb-10"
```
Comment (badabam) [inline index.html:62]: Looks like you do too much `absolute` positioning for me. Will be a topic next week, let's talk about it then.

Comment (badabam) [reply index.html:62]: For now you can keep this and merge – it's not "wrong", but maybe not "optimal".

**Review by badabam — APPROVED** [review]
Great work! I can see your experience and you do a lot of things in a good way. At first glance, your usage of `absolute` positioning looks unnecessary. We can talk about it next week. You can ship this!

File: index.html (hunk 1 above)
Comment (F-Kirchhoff) [reply index.html:30]: its a special split menu i came up with, not sure if ill keep it. its there so both parts are positioned centered but left sided and right sided.

File: index.html (hunk 2 above)
Comment (F-Kirchhoff) [reply index.html:31]: mainly design and I wanted it to do differently then the times before

File: index.html (hunk 3 above)
Comment (F-Kirchhoff) [reply index.html:14]: I want to make sure that on fullscreen only a portion of the screen is used. maybe there are more elegant ways?

File: index.html (hunk 6 above)
Comment (F-Kirchhoff) [reply index.html:62]: It was indeed tricky to get the postioning as I wanted it to. But I would love a more simple solution!

**Pull request merged**